ddtrace/settings/config.py

```diff
@@ -242,3 +243,10 @@ def __repr__(self):
         cls = self.__class__
         integrations = ", ".join(self._config.keys())
         return "{}.{}({})".format(cls.__module__, cls.__name__, integrations)
+
+    @deprecated(
+        message="HttpServerConfig has been renamed to _HttpServerConfig and is no longer be part of the public API.",
```

Comment:
```suggestion
        message="HttpServerConfig will be removed",
```

I don't think we need to state that it was renamed or removed from the API, we can just state that it is going away, since that is true from customer's perspective.